{"name": "Создание заказа с авторизацией при неверном хеше ингредиентов", "status": "passed", "start": 1723625224006, "stop": 1723625224262, "uuid": "36e76db3-7b77-4eb0-b6fa-0079f48d2d3a", "historyId": "fbd144d9ed6216a1bfa3538044a15efc", "testCaseId": "fbd144d9ed6216a1bfa3538044a15efc", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_auth_wrong_hash_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "Hapuga"}, {"name": "thread", "value": "852-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}